userguide/tasks.rst

```diff
@@ -1277,54 +1165,17 @@ in data.
    new replicated data can be created.
 
 
-Navigating to
-:menuselection:`Tasks --> Replication Tasks` displays
-:numref:`Figure %s <zfs_repl_task_list_fig>`, the list of
-replication tasks. :guilabel:`Status` shows the current status of each
-replication task. The display is updated periodically, always showing
-the latest status.
-
-.. _zfs_repl_task_list_fig:
-
-.. figure:: images/tasks-replication-tasks.png
-   :width: 90%
-
-   Replication Task List
-
-
-.. note:: The encryption key that was copied from the source computer
-   (*Alpha*) to the destination computer (*Beta*) is an RSA public
-   key located in the :file:`/data/ssh/replication.pub` file on the
-   source computer. The host public key used to identify the
-   destination computer (*Beta*) is from the
-   :file:`/etc/ssh/ssh_host_rsa_key.pub` file on the destination
-   computer.
-
-
-.. _Replication Encryption:
-
-Replication Encryption
-~~~~~~~~~~~~~~~~~~~~~~
-
-The default :guilabel:`Encryption Cipher` *Standard* setting provides
-good security. *Fast* is less secure than *Standard* but can give
-reasonable transfer rates for devices with limited cryptographic
-speed. For networks where the entire path between source and
-destination computers is trusted, the *Disabled* option can be chosen
-to send replicated data without encryption.
-
-
 .. _Limiting Replication Times:
 
 Limiting Replication Times
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The :guilabel:`Begin` and :guilabel:`End` times in a replication task
-make it possible to restrict when replication is allowed. These times
-can be set to only allow replication after business hours, or at other
-times when disk or network activity will not slow down other
-operations like snapshots or :ref:`Scrub Tasks`. The default settings
-allow replication to occur at any time.
+The :guilabel:`Schedule` :guilabel:`Begin` and :guilabel:`End` times in
```

Comment:
```suggestion
The :guilabel:`Schedule`, :guilabel:`Begin`, and :guilabel:`End` times in
```

Reply:
I think that Begin and End are part of the Schedule, not three different items.

Reply:
They are related items, but appear as individual fields. Could go either way on this one.